lib/pensio_api/altapay_version.rb

```diff
@@ -0,0 +1,10 @@
+module PensioAPI
+
+    class AltapayVersion
+
+        def self.getVersion
```

Comment:
As commented before, this should be a constant, not a method.

Reply:
Actually I was merging on another repository and it come by mistake. Closed pull request already. Thank you.